components/buttons/buttons.css

```diff
@@ -0,0 +1,179 @@
+/* ---default button */
+.btn {
+    border: none;
+    background-color: var(--ICONS-ORANGE);
+    color: var(--pure-white);
+    padding: 1.5rem 2rem;
+    cursor: pointer;
+    font-weight: bold;
+    transition: all .5s ease-in-out;
+}
+
+/* PRIMARY BUTTON */
+.btn-pri {
+    background-color: var(--495057);
+    color: var(--pure-white);
+}
+
+.btn-pri:hover {
+    background: var(--6C757D);
+}
+
+/* SECONDARY BUTTON */
+.btn-sec {
+    background-color: var(--DEE2E6);
+    color: var(--495057);
+}
+
+.btn-sec:hover {
+    background-color: var(--DE2EE6);
+}
+
+/* ERROR BUTTON */
+.btn-error {
+    background-color: var(--ERROR-RED);
+}
+
+.btn-error:hover {
+    background: var(--ERROR-RED-DARK);
+}
+
+/* BUTTON SUCCESS */
+.btn-success {
+    background: var(--SUCCESS-GREEN);
+}
+
+.btn-success:hover {
+    background: var(--SUCCESS-GREEN-DARK);
+}
+
+/* BUTTON OUTLINE */
+.btn.outline {
+    color: var(--ICONS-ORANGE);
+    border: solid var(--ICONS-ORANGE) 2px;
+    background-color: var(--pure-white);
+    border-radius: 4px;
+    padding: 1rem 2rem;
+}
+
+.btn.outline:hover,
+.btn.outline:focus {
+    background-color: var(--ICONS-ORANGE);
+    color: var(--pure-white);
+}
+
+.btn.outline-pri {
+    color: var(--PURE-BLACK);
+    border: solid var(--PURE-BLACK) 2px;
+    background-color: var(--pure-white);
+    border-radius: 4px;
```

Comment:
since you've used border-radius: 4px many a times in your code, you can make a class and add that class to the elements.

Reply:
Thanks for reviewing this I will do this for sure sai